packages/routes/routes.0.6.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "Anurag Soni <anurag@sonianurag.com>"
+authors: [ "Anurag Soni <anurag@sonianurag.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/anuragsoni/routes"
+bug-reports: "https://github.com/anuragsoni/routes/issues"
+dev-repo: "git+https://github.com/anuragsoni/routes.git"
+doc: "https://anuragsoni.github.io/routes/"
+tags: [ "router" "http" ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" { >= "1.1" }
```

Comment:
Are you sure that `(using fmt 1.1)` works with dune 1.1 and does not require  1.5?

Reply:
I think so. My understanding was that the version needs to match the dune language version in the dune-project file. I’ll check the docs again to confirm

Reply:
Currently dune extensions do not check for a matching dune version number (see https://github.com/ocaml/dune/issues/2957) and according to https://github.com/ocaml/dune/pull/1821, fmt 1.1 has been added in dune.1.7.0. So I believe your package requires at least dune 1.7

Reply:
Thank you for sharing those links. I think if I need to change the dune version constraint, I’ll go over the dune docs to see if there are any other extension that I’d like to use. Or see if I stop using the fmt extension. I’ll close this pull request for now and revisit this once I’ve figured out which dune extensions I need.